### PR TITLE
Added series empty subtitle episodes progress bar labels

### PR DESCRIPTION
**Pull request by ilarramendi**
Before:
image
After:
image

Im not 100% sure if i should have placed styles there, i can move them if needed!

### CODE REVIEW

File: frontend/src/assets/_bazarr.scss [hunk 1]
```diff
@@ -52,6 +52,17 @@ $header-height: 64px;
...
+  .table-progress-label {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    color: rgb(232, 230, 227);
+    line-height: 1.55;
+    font-size: calc(1rem * 0.65);
+    font-weight: 700;
+  }
```
Comment (anderson-oki) [inline frontend/src/assets/_bazarr.scss:64]: This should no be in `_bazarr.scss` as it is not global.

You should instead use styles API of mantine.

https://mantine.dev/core/progress/?t=styles-api

<Progress.label styles={{ .mantine-Progress-label: ...}} /`.

**Review by anderson-oki — CHANGES_REQUESTED** [review]

**Comment by anderson-oki** [comment]
The idea is pretty good, i was thinking on implement this. The implementation needs some adjustments tho! Thank you!

File: frontend/src/assets/_bazarr.scss [hunk 2]
```diff
@@ -52,6 +52,17 @@ $header-height: 64px;
     overflow-wrap: anywhere;
   }
 
+  .table-progress-label {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    color: rgb(232, 230, 227);
```
Comment (anderson-oki) [inline frontend/src/assets/_bazarr.scss:60]: Do not use absolute colors, use mantine color variables.

File: frontend/src/assets/_bazarr.scss [hunk 3]
```diff
@@ -52,6 +52,17 @@ $header-height: 64px;
...
+    line-height: 1.55;
+    font-size: calc(1rem * 0.65);
```
Comment (anderson-oki) [inline frontend/src/assets/_bazarr.scss:62]: Use mantine font size variables

File: frontend/src/assets/_bazarr.scss (hunk 1 above)
Comment (ilarramendi) [reply frontend/src/assets/_bazarr.scss:64]: Im not sure if i understand, something like this?
```js
              {episodeMissingCount === episodeFileCount && <Progress.Label style={{
                position: 'absolute',
                top: '3px',
                left: '50%',
                transform: 'translateX(-50%)'
              }}>{label}</Progress.Label>}
```
This looks the same as above, should i use a scss module for those inline styles?

Comment (anderson-oki) [reply frontend/src/assets/_bazarr.scss:64]: No this is wrong. You should use `styles` and not style.

Comment (anderson-oki) [reply frontend/src/assets/_bazarr.scss:64]: Let me pick an example to you.

Comment (anderson-oki) [reply frontend/src/assets/_bazarr.scss:64]: Check `pages/views/ItemOverview.tsx`

Comment (ilarramendi) [reply frontend/src/assets/_bazarr.scss:64]: Somethink like this is ok? Or should i move it to a module.scss like in that example?
```js
{episodeMissingCount === episodeFileCount && <Progress.Label styles={{
  label: {
    position: 'absolute',
    top: '3px',
    left: '50%',
    transform: 'translateX(-50%)'
  }
}}>{label}</Progress.Label>}
```
Sorry, I never used Mantine before

Comment (anderson-oki) [reply frontend/src/assets/_bazarr.scss:64]: You can leave inline for now, I will refactor the modules properly later, now the series filename is called index.tsx, that will cause issues creating a module css for that.

Comment (anderson-oki) [reply frontend/src/assets/_bazarr.scss:64]: > Somethink like this is ok? Or should i move it to a module.scss like in that example?

Your example looks good for now! Moving to a module.css might have more complications, i will work on refactor this on the future.

Comment (ilarramendi) [reply frontend/src/assets/_bazarr.scss]: pushed, thanks

**Comment by anderson-oki** [comment]
@ilarramendi 

I recommend you install the hooks that will lint your code before pushing by using `npm run prepare`. Or you can format manually or you can run `npm run format`.

**Comment by ilarramendi** [comment]
@anderson-oki Hooks may not bee working correctly for linux

**Pull request merged**